db/schema.rb

```diff
@@ -0,0 +1,65 @@
+# encoding: UTF-8
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# Note that this schema.rb definition is the authoritative source for your
+# database schema. If you need to create the application database on another
+# system, you should be using db:schema:load, not running all the migrations
+# from scratch. The latter is a flawed and unsustainable approach (the more migrations
+# you'll amass, the slower it'll run and the greater likelihood for issues).
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 20160114171839) do
+
+  create_table "contents", force: :cascade do |t|
+    t.string   "content_id"
+    t.string   "text"
+    t.string   "create_time"
+    t.integer  "favorites"
+    t.string   "embed_code"
+    t.integer  "retweet_count"
+    t.datetime "created_at",    null: false
+    t.datetime "updated_at",    null: false
```

Comment:
Note the "created_at" and "updated_at" columns that are automatically generated and maintained.